SharedPackages/BrowserServicesKit/Sources/Subscription/Flows/Stripe/StripePurchaseFlowV2.swift

```diff
@@ -26,11 +26,21 @@ import PixelKit
 public enum StripePurchaseFlowError: DDGError {
     case noProductsFound
     case accountCreationFailed(Error)
+    case apiCallFailed(Error)
+    case emptyProductsFromAPI
+    case emptyAfterFiltering
+    case tierCreationFailed
+    case invalidProductData
 
     public var description: String {
         switch self {
         case .noProductsFound: "No products found."
         case .accountCreationFailed(let error): "Account creation failed: \(error)"
+        case .apiCallFailed(let error): "API call failed: \(error)"
+        case .emptyProductsFromAPI: "API returned empty products."
+        case .emptyAfterFiltering: "No products after filtering."
+        case .tierCreationFailed: "Failed to create tiers."
+        case .invalidProductData: "Invalid product data."
```

Comment:
Please double check but from what I see `StripePurchaseFlowError.invalidProductData` is not used in any place.